WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -312,7 +328,7 @@ async Task<(AliceClient? AliceClient, PersonCircuit? PersonCircuit)> RegisterInp
 				{
 					await Task.Delay(delay, cancellationToken).ConfigureAwait(false);
 				}
-				return await RegisterInputAsync(coin, cancellationToken).ConfigureAwait(false);
+				return await RegisterInputAsync(coin).ConfigureAwait(false);
```

Comment:
Because the `RegisterInput` method calls another method called `CreateRegisterAndConfirmInputAsync` that as its name says it does not only one thing but two instead (Register and Confirm) and for that reason it required or well a refactoring or, as i did here, to pass two cancellation tokens so, instead of passing the cancellation tokens as parameters, now it uses the CTs in the outer scope.

We can pass them anyway. Let me know if you think passing them as parameters is better and why.